src/interface.jl

```diff
@@ -324,13 +401,14 @@ logabsdetjac(b::Scale, x) = log(abs(b.a))
 # Simplex bijector #
 ####################
 struct SimplexBijector{T} <: Bijector where {T} end
+SimplexBijector() = SimplexBijector{Val{true}}()
 
 const simplex_b = SimplexBijector{Val{false}}()
 const simplex_b_proj = SimplexBijector{Val{true}}()
```

Comment:
Is this still used?

Reply:
Not sure? I thought it was nice to have due to this basically corresponding to the default `link` and `invlink`.

Reply:
Should maybe also include a `SimplexBijector(proj::bool) = SimplexBijector{Val{proj}}()`?

Reply:
Looks like a good constructor singature to me.

Reply:
I've also removed the `simplex_b` and `simplex_b_proj` as they are no longer used (Maybe this was actually what you were asking but for some reason Github showed the `SimplexBijector()` line as the one you were referring to)